### ISSUE TITLE
Issues with packages on Debian/Ubuntu based systems

### ISSUE THREAD

**Issue by Nolkaloid** [CE-Programming/CEmu#351]
- Repositories are broken for the Ubuntu packages (at least). The key seems outdated.
- And when installing the `.deb` directly, there is no menu entry created.

Otherwise thanks for this amazing project!

**Comment by adriweb** [comment]
Yes I noticed the outdated key issue but I doubt there is anything we can do ourselves  about that since it's handled by the build server...

Regarding the menu icon, I think it's more or less supposed to be handled somehow but I don't manage that part myself. Can someone weigh in? Might be a configuration issue.

**Comment by adriweb** [comment]
Seems solved upstream (OpenSuse Build Service) although we haven't released anything new since v2.0
